couchdb/client.py

```diff
@@ -297,9 +295,13 @@ def verify_user(self, token_or_name, password=None):
             else:
                 header = {
                     'Accept': 'application/json',
-                    'Authorization': 'Basic ' + strip(encodestring(token_or_name + ':' + password)),
+                    'Content-Type': 'application/json',
                 }
-                status, _, _ = self.resource.post_json('_session', headers=header)
+                body = {
+                    'name': token_or_name,
+                    'password': password,
+                }
+                status, _, _ = self.resource.post_json('_session', headers=header, body=body)
```

Comment:
I don't think the `headers=` keyword argument needs to be explicitly named everywhere.